feat(interface management): add section to manage your services (#7497)
ovh/docs: pages/public_cloud/compute/03-public-cloud-interface-walk-me/guide.fr-fr.md

|**Contacts and Rights**|Outre la possibilité de changer le contact technique ou le contact de facturation de votre projet, vous aurez la possibilité d'[ajouter d'autres contacts](/pages/public_cloud/compute/change_project_contacts) (compte OVHcloud) pour administrer techniquement votre projet. Vous pouvez également ajouter des utilisateurs en consultation uniquement (« *read-only* »).|
|**Project settings**|Ce dernier outil vous permet de configurer les paramètres généraux du projet comme son nom, sa configuration en tant que « projet par défaut du compte », la compatibilité HDS, ou encore de [supprimer votre projet Public Cloud](/pages/public_cloud/compute/delete_a_project)|

### Gestion des services

> > [!info]

Reviewer: ```suggestion
> [!primary]
```

>

> > Dans cette section, nous vous présentons un aperçu des différentes options de gestion des services proposées par OVHcloud, à travers trois outils principaux : l'espace client OVHcloud, Horizon et l'Openstack API. Chacun de ces outils a été conçu pour répondre à des besoins spécifiques en fonction du niveau d'expertise, des préférences en termes de gestion, ainsi que des exigences de performance et de personnalisation.

Reviewer: ```suggestion
> Dans cette section, nous vous présentons un aperçu des différentes options de gestion des services proposées par OVHcloud, à travers trois outils principaux : l'espace client OVHcloud, Horizon et l'API OpenStack. Chacun de ces outils a été conçu pour répondre à des besoins spécifiques en fonction de votre niveau d'expertise, de vos préférences en termes de gestion, ainsi que de vos exigences de performance et de personnalisation.
```

>
> La matrice ci-dessous compare les caractéristiques clés de chaque outil afin de vous aider à choisir la solution la mieux adaptée à vos besoins. Que vous soyez débutant, utilisateur intermédiaire ou expert en automatisation, cette comparaison vous permettra de mieux comprendre les avantages, la facilité d'utilisation, les niveaux de compétence requis, et la scalabilité de chaque outil.
>

> | Critères/Caractéristiques                   | espace client OVHcloud | Horizon | Openstack API |

Reviewer: ```suggestion
| Critères/Caractéristiques                   | Espace client OVHcloud | Horizon | Openstack API |
```

| Performances et scalabilité                 | Performances limitées et scalabilité basique. Adapté aux déploiements de petite taille ou non critiques. La scalabilité est généralement manuelle et plus lente. Idéal pour des environnements statiques ou des petits projets. | Performances moyennes avec une gestion améliorée de la scalabilité via l'interface graphique. Scalabilité plus rapide qu'avec Manager, mais limitée par l’interface. Adapté aux projets de taille moyenne ou nécessitant une certaine évolutivité. | Performances optimales et scalabilité complète. Permet des déploiements massifs, automatisés et rapides via des scripts ou des outils tiers. Idéal pour des infrastructures dynamiques, des charges lourdes et des environnements nécessitant une haute élasticité. Recommandé pour des architectures critiques. |

> | Pèrimètres d'utilisation (Compute)          | - Création et gestion simplifiée des machines virtuelles (VM).<br> - Redimensionnement des VM après création (modification du modèle flavor à chaud ou à froid, selon les ressources disponibles).<br> - Sélection de configurations standards pour les VM (RAM, CPU, stockage).<br> - Gestion des actions essentielles : démarrage, arrêt et suppression des VM.<br> - Accès aux snapshots pour des sauvegardes rapides et restaurations simplifiées.<br> - Attribution et gestion des adresses IP flottantes.<br> - Création et administration basique des disques additionnels.<br> - Supervision des ressources avec un monitoring essentiel (CPU, mémoire, stockage). | - Gestion avancée des accès : prise en charge du contrôle des accès basé sur les rôles (RBAC) pour une gestion multi-utilisateurs sécurisée.<br> - Administration réseau avancée : création et gestion de réseaux privés complexes associés aux VM (réseaux internes, sous-réseaux).<br> - Déploiement de VM avec configurations réseau spécifiques, incluant la gestion de plusieurs interfaces réseau.<br> - Utilisation d'images personnalisées pour la création de VM, en alternative aux images standards proposées par OVHcloud.<br> - Intégration de workflows préconfigurés via Horizon pour automatiser les déploiements et configurations. | - Automatisation complète : toutes les actions disponibles dans Manager et Horizon sont réalisables via l’API, avec la possibilité de les automatiser et de les scripter.<br> - Déploiement d’infrastructures en mode Infrastructure as Code (IaC) à l’aide d’outils comme Terraform, Ansible ou de scripts personnalisés.<br> - Intégration aux pipelines CI/CD pour des déploiements automatisés (ex. intégration avec GitLab CI).<br> - Gestion avancée des quotas de ressources (nombre de CPU, RAM total, etc.).<br> - Scalabilité dynamique : ajustement automatique des instances en fonction de la charge via API ou scripts.<br> - Monitoring et collecte de métriques personnalisées via API, offrant plus de granularité que les interfaces Horizon ou Manager. |

Reviewer: ```suggestion
| Périmètres d'utilisation (Compute)          | - Création et gestion simplifiée des machines virtuelles (VM).<br> - Redimensionnement des VM après création (modification du modèle flavor à chaud ou à froid, selon les ressources disponibles).<br> - Sélection de configurations standards pour les VM (RAM, CPU, stockage).<br> - Gestion des actions essentielles : démarrage, arrêt et suppression des VM.<br> - Accès aux snapshots pour des sauvegardes rapides et restaurations simplifiées.<br> - Attribution et gestion des adresses Floating IP.<br> - Création et administration basique des disques additionnels.<br> - Supervision des ressources avec un monitoring essentiel (CPU, mémoire, stockage). | - Gestion avancée des accès : prise en charge du contrôle des accès basé sur les rôles (RBAC) pour une gestion multi-utilisateurs sécurisée.<br> - Administration réseau avancée : création et gestion de réseaux privés complexes associés aux VM (réseaux internes, sous-réseaux).<br> - Déploiement de VM avec configurations réseau spécifiques, incluant la gestion de plusieurs interfaces réseau.<br> - Utilisation d'images personnalisées pour la création de VM, en alternative aux images standards proposées par OVHcloud.<br> - Intégration de workflows préconfigurés via Horizon pour automatiser les déploiements et configurations. | - Automatisation complète : toutes les actions disponibles dans l'espace client OVHcloud et Horizon sont réalisables via l’API, avec la possibilité de les automatiser et de les scripter.<br> - Déploiement d’infrastructures en mode Infrastructure as Code (IaC) à l’aide d’outils comme Terraform, Ansible ou de scripts personnalisés.<br> - Intégration aux pipelines CI/CD pour des déploiements automatisés (ex. intégration avec GitLab CI).<br> - Gestion avancée des quotas de ressources (nombre de CPU, RAM total, etc.).<br> - Scalabilité dynamique : ajustement automatique des instances en fonction de la charge via API ou scripts.<br> - Monitoring et collecte de métriques personnalisées via API, offrant plus de granularité que l'interface Horizon ou l'espace client OVHcloud. |
```

> | Pèrimètres d'utilisation (Network)          | - Création et gestion de réseaux privés (Private Networks).<br> - Association d'adresses IP flottantes (Floating IP) et d'adresses IP additionnelles.<br> - Configuration de routage de base (Basic Routing). | - Gestion avancée des règles de sécurité via les groupes de sécurité (Security Groups).<br> - Visualisation des topologies réseau pour une gestion simplifiée.<br> - Support complet des sous-réseaux IPv6 pour une connectivité moderne.<br> - Configuration des politiques QoS (Quality of Service) pour prioriser les ressources réseau. | - Accès à toutes les fonctionnalités disponibles sur Horizon et le Manager.<br> - Création de routes personnalisées pour une gestion réseau plus flexible.<br> - Configuration précise des politiques QoS (Quality of Service).<br> - Gestion avancée des VRRP (Virtual Router Redundancy Protocol) pour assurer la redondance des routeurs.<br> - Automatisation des actions réseau grâce à des scripts (Infrastructure as Code).<br> - Intégration avec des solutions SDN (Software-Defined Networking) pour une gestion réseau agile. |

Reviewer: ```suggestion
| Périmètres d'utilisation (Network)          | - Création et gestion de réseaux privés (Private Networks).<br> - Association d'adresses Floating IP et d'adresses Additional IP.<br> - Configuration de routage de base (Basic Routing). | - Gestion avancée des règles de sécurité via les groupes de sécurité (Security Groups).<br> - Visualisation des topologies réseau pour une gestion simplifiée.<br> - Support complet des sous-réseaux IPv6 pour une connectivité moderne.<br> - Configuration des politiques QoS (Quality of Service) pour prioriser les ressources réseau. | - Accès à toutes les fonctionnalités disponibles sur Horizon et l'espace client OVHcloud.<br> - Création de routes personnalisées pour une gestion réseau plus flexible.<br> - Configuration précise des politiques QoS (Quality of Service).<br> - Gestion avancée des VRRP (Virtual Router Redundancy Protocol) pour assurer la redondance des routeurs.<br> - Automatisation des actions réseau grâce à des scripts (Infrastructure as Code).<br> - Intégration avec des solutions SDN (Software-Defined Networking) pour une gestion réseau agile. |
```

> | Pèrimètres d'utilisation (Storage & Backup) | - Création et gestion des volumes de stockage : Object Storage, Block Storage, et File Storage.<br> - Sauvegarde automatique basique (snapshots) des volumes, avec possibilité de restauration.<br> - Association des volumes de stockage aux instances pour un accès simplifié.<br> - Gestion des conteneurs Object Storage (Swift) pour organiser les données.<br> - Visualisation de l'état des volumes et de l'espace de stockage utilisé.<br> - Ajout et gestion de fichiers dans un Object Storage. | - Gestion avancée des snapshots : rétention, duplication, et autres options pour un contrôle précis des sauvegardes.<br> - Gestion détaillée des partages de volumes (multi-attach) pour une flexibilité accrue.<br> - Création et gestion des sauvegardes planifiées avec des politiques de sauvegarde personnalisables.<br> - Surveillance de l'utilisation des quotas de stockage pour un suivi optimal de l'espace disponible. | - Fonctionnalités accessibles via Horizon et Manager.<br> - Intégration et automatisation via scripts (Infrastructure as Code) pour une gestion fluide.<br> - Configuration avancée des partages réseau (NFS, CIFS) pour une flexibilité accrue dans l'organisation des fichiers.<br> - Gestion précise des métadonnées des objets dans Object Storage pour un contrôle optimal des données.<br> - Configuration avancée de la réplication et du versioning d'objets pour une haute disponibilité et une gestion des versions complète.<br> - Accès direct à l'API Swift pour une intégration fluide avec des outils tiers.<br> - Création de workflows personnalisés pour automatiser et gérer efficacement les processus de sauvegarde. |

Reviewer: ```suggestion
| Périmètres d'utilisation (Storage & Backup) | - Création et gestion des volumes de stockage : Object Storage, Block Storage et File Storage.<br> - Sauvegarde automatique basique (snapshots) des volumes, avec possibilité de restauration.<br> - Association des volumes de stockage aux instances pour un accès simplifié.<br> - Gestion des conteneurs Object Storage (Swift) pour organiser les données.<br> - Visualisation de l'état des volumes et de l'espace de stockage utilisé.<br> - Ajout et gestion de fichiers dans un Object Storage. | - Gestion avancée des snapshots : rétention, duplication et autres options pour un contrôle précis des sauvegardes.<br> - Gestion détaillée des partages de volumes (multi-attach) pour une flexibilité accrue.<br> - Création et gestion des sauvegardes planifiées avec des politiques de sauvegarde personnalisables.<br> - Surveillance de l'utilisation des quotas de stockage pour un suivi optimal de l'espace disponible. | - Fonctionnalités accessibles via Horizon et l'espace client OVHcloud.<br> - Intégration et automatisation via scripts (Infrastructure as Code) pour une gestion fluide.<br> - Configuration avancée des partages réseau (NFS, CIFS) pour une flexibilité accrue dans l'organisation des fichiers.<br> - Gestion précise des métadonnées des objets dans Object Storage pour un contrôle optimal des données.<br> - Configuration avancée de la réplication et du versioning d'objets pour une haute disponibilité et une gestion des versions complète.<br> - Accès direct à l'API Swift pour une intégration fluide avec des outils tiers.<br> - Création de workflows personnalisés pour automatiser et gérer efficacement les processus de sauvegarde. |
```
